packages/react/src/components/Header/header.scss

```diff
@@ -20,24 +20,28 @@
   border-left: none;
   border-right: none;
   border-top: none;
-  background-color: var(--oxygen-palette-background-paper);
 
   &-collapsible-hamburger {
     display: flex;
     align-items: center;
     justify-content: flex-start;
-    padding: 0 12px;
+    padding: 0 12px 0 23px;
   }
 
   .oxygen-brand {
     display: flex;
     align-items: center;
+    width: calc(var(--oxygen-customComponents-Navbar-properties-width) - 77px);
```

Comment:
Why did we add the width like this?

Reply:
This is to make sure the logo covers the entire width of the navbar.